src/Nancy/ErrorHandling/DefaultStatusCodeHandler.cs

```diff
@@ -62,47 +66,60 @@ public void Handle(HttpStatusCode statusCode, NancyContext context)
                 return;
             }
 
-            string errorPage;
-
-            if (!this.errorPages.TryGetValue(statusCode, out errorPage))
+            if (!this.errorMessages.ContainsKey(statusCode) || !this.errorPages.ContainsKey(statusCode))

```

Comment:
Is this condition correct? We bail out if the `statusCode` is not in both dictionaries. Is that right?